scripts/ty_benchmark/README.md

```diff
@@ -7,15 +7,24 @@
 
 1. Build ty: `cargo build --bin ty --release`
 1. `cd` into the benchmark directory: `cd scripts/ty_benchmark`
+1. Install Pyright: `npm install`
 1. Run benchmarks: `uv run benchmark`
 
+Requires hyperfine 1.20 or newer.
+
 ## Known limitations
 
-ty only implements a tiny fraction of Mypy's and Pyright's functionality,
-so the benchmarks aren't in any way a fair comparison today. However,
-they'll become more meaningful as we build out more type checking features in ty.
+The tested type checkers implement Python's type system to varying degrees and
+some projects only successfully pass type checking using a specific type checker.
 
 ### Windows support
 
 The script should work on Windows, but we haven't tested it yet.
 We do make use of `shlex` which has known limitations when using non-POSIX shells.
+
+## Updating the benchmark
+
+The benchmark script supports snapshoting the results when running with `--snapshot` and `--accept`.
+The goal of those snapshots is to catch accidental regressions. They are not intended
+as a testing tool. E.g. the snapshot runner doesn't account for platform differences so that
```

Comment:
Can/should we "pin" the platform to an arbitrary one, i.e. pass the equivalent of ty's `--python-platform=linux` to all type checkers?

Reply:
It's tricky because you know Python. 

Solving for one platform requires installing the dependency for that platform, and that failed for me at least for some of the projects on macos.